requirements.txt

```diff
@@ -1,4 +1,4 @@
-django >= 1.11.5
+django==1.11.7
```

Comment:
Can we change this to be the highest 1.11.x instead? I do not know of any team member who watched their dist mailing lists.

Reply:
Sure, looks like their policy is safe for 1.11.x tracking. https://docs.djangoproject.com/en/2.0/internals/release-process/#official-releases